zkevm-circuits/src/evm_circuit/execution/error_invalid_jump.rs

```diff
@@ -80,39 +70,31 @@ impl<F: Field> ExecutionGadget<F> for ErrorInvalidJumpGadget<F> {
         });
 
         // Look up bytecode length
-        let code_len = cb.query_cell();
         cb.bytecode_length(cb.curr.state.code_hash.expr(), code_len.expr());
```

Comment:
Aren't we missing the lookup to the bytecode to verify `value` and `is_code`?

Reply:
It seems that the bytecode is lookup when destination is in valid range in [the code below](https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/a3226b4d3c55f61a7cd9ef1258f6b7fc8d5c9d41/zkevm-circuits/src/evm_circuit/execution/error_invalid_jump.rs#L77).

Reply:
thanks, you're right!